domain/domain.go

```diff
@@ -1587,13 +1588,15 @@ func (do *Domain) updateStatsWorker(ctx sessionctx.Context, owner owner.Manager)
 	dumpFeedbackTicker := time.NewTicker(200 * lease)
 	loadFeedbackTicker := time.NewTicker(5 * lease)
 	dumpColStatsUsageTicker := time.NewTicker(100 * lease)
+	goTunerTicker := time.NewTicker(3 * time.Second)
```

Comment:
Why is it 3 seconds? I think `cgroup.InContainer()` shouldn't change after the process started.